helpers/repositories.json

```diff
@@ -395,6 +395,7 @@
    "bigmandave/doge-node",
    "treyyoder/quakejs:latest",
    "biosmarcel/scribble.rs:latest",
+   "fegauthier/persistent-storage-test:latest"
```

Comment:
Hi, can you please add comma here and ideally move it to the end of the array?

Reply:
fegauthier/persistent-storage-test:latest was already there. I just removed the duplicate that I commited.